docs/using/recipes.md

```diff
@@ -516,3 +516,24 @@ RUN pip install jupyter_contrib_nbextensions && \
 
 Ref:
 [https://github.com/jupyter/docker-stacks/issues/675](https://github.com/jupyter/docker-stacks/issues/675)
+
+## Enable auto-sklearn notebooks
+
+Using auto-sklearn requires swig, which the other notebook images lack, so it cant be experimented with. Also, there is no Conda package for auto-sklearn.
```

Comment:
Use inline code for `auto-sklearn`, `swig`

Reply:
Added, thanks!